docs/docker.md

```diff
@@ -2,15 +2,33 @@
 
 ## Overview
 
-Docker can automate much of the configuration for you.
+Docker packages software into virtual containers so it can run on any Linux, Windows, or macOS computer. It can automate much of the configuration for you.
 
-
-## Using docker
+## Using Docker
 
 ### Prerequisites
 
-* Recent versions of `docker` and `docker-compose`.
-* A database dump of the full AGAGD schema, stored at `schema.sql`. (in the root directory of the repository)
+- Recent versions of `docker` and `docker-compose`.
+- A database dump of the full AGAGD schema, stored at `schema.sql`. (in the root directory of the repository)
```

Comment:
Is there a specific reason for the change of `*` to `-`?

Reply:
My markdown linter automatically changed the * to -

Reply:
I have added Linter in https://github.com/usgo/agagd/pull/201, but these may change to be more explicit about how our Markdown is addressed. The configuration is in: https://github.com/usgo/agagd/blob/main/.agagd_mdlrc_style.rb.